cache/lru_cache.cc

```diff
@@ -263,10 +297,24 @@ void LRUCacheShard::MaintainPoolSize() {
     lru_low_pri_ = lru_low_pri_->next;
     assert(lru_low_pri_ != &lru_);
     lru_low_pri_->SetInHighPriPool(false);
+    lru_low_pri_->SetInLowPriPool(true);
     size_t total_charge =
         lru_low_pri_->CalcTotalCharge(metadata_charge_policy_);
     assert(high_pri_pool_usage_ >= total_charge);
     high_pri_pool_usage_ -= total_charge;
+    low_pri_pool_usage_ += total_charge;
+  }
+
+  while (low_pri_pool_usage_ > low_pri_pool_capacity_) {
+    // Overflow last entry in low-pri pool to bottom-pri pool.
+    lru_bottom_pri_ = lru_bottom_pri_->next;
+    assert(lru_bottom_pri_ != &lru_);
```

Comment:
should it be lru_low_pri_ for lru_?

Reply:
Nope, same as upstream https://github.com/facebook/rocksdb/pull/10461/files#diff-1b304e22abe7464a3675347f930d19e35a77e12f9e8d590a9e5c79dd7f61398bR309